interfaces/builtin/hugepages_control.go

```diff
@@ -42,6 +42,9 @@ const hugepagesControlConnectedPlugAppArmor = `
 /sys/devices/system/node/node[0-9]*/hugepages/{,hugepages-[0-9]*}/nr_hugepages w,
 @{PROC}/sys/vm/nr_{hugepages,hugepages_mempolicy,overcommit_hugepages} rw,
 
+# Allow mounting hugetlbfs
+mount fstype=hugetlbfs,
```

Comment:
can you show what sort of denials you see with this? we try very hard to restrict mounts as much as possible with options or locations, since they are usually rife with sandbox escape mechanisms

Reply:
err rather what sort of denials you see without this?

Reply:
@anonymouse64 , thanks for review. Below is the message we got from the `snappy-debug` without this mount permission:
```
= AppArmor =
Time: May 22 01:27:14
Log: apparmor="DENIED" operation="mount" info="failed srcname match" error=-13 profile="snap.acrn.launch-uos" name="/var/snap/acrn/common/run/hugepage/huge_lv1/vm-clear-34500-kvm/" pid=1548 comm="acrn-dm" fstype="hugetlbfs" srcname="none"
```
From the ACRN source code, the program failed at [init_hugetlb()](https://github.com/projectacrn/acrn-hypervisor/blob/master/devicemodel/core/main.c#L1015), and bisect the issue indicates it was caused by this [mount](https://github.com/projectacrn/acrn-hypervisor/blob/master/devicemodel/core/hugetlb.c#L443) operation. So we'd like to add hugetlbfs mount permission to the `hugepages-control` interface, because it should be hugepages MMIO related. Thanks.

Reply:
We should further restrict this rule to specify the srcname as `none` and the destination (`name`) to something like $SNAP_DATA or $SNAP_COMMON 

```suggestion
mount fstype=hugetlbfs none -> /var/snap/{@{SNAP_NAME},@{SNAP_INSTANCE_NAME}}/** ,
```

Reply:
@anonymouse64 , thanks for coach, I've incorporated the suggested change to the commit.